### ISSUE TITLE
Consistancy between files extensions

### ISSUE THREAD

**Issue by apsylone** [phpmentors-jp/docker-images#27]
I think it will be better to have bash scripts ending with ".sh" extension for the filename just like some.

Maybe it is some reasons behind this ? 
"init" scripts doesn't have any extensions compare to "locale.sh" for example.

**Comment by iteman** [comment]
Yes, while the scripts ending with `.sh` extension are libraries, other scripts are executable.
